src/woo_publications/contrib/documents_api/tests/test_client.py

```diff
@@ -152,3 +154,17 @@ def test_can_unlock_document_after_uploads_completed(self):
         self.assertFalse(detail_data["locked"])
         bestandsdelen = detail_data["bestandsdelen"]
         self.assertEqual(len(bestandsdelen), 0)
+
+    @requests_mock.Mocker()
+    @override_settings(LANGUAGE_CODE="en-en")
+    def test_destroy_document_error(self, m):
```

Comment:
you can now the VCR helper: https://maykin-django-common.readthedocs.io/en/latest/reference/vcr.html#maykin_common.vcr.VCRMixin.vcr_raises instead of requests_mock I believe